runner/job/task.go

```diff
@@ -0,0 +1,56 @@
+package job
+
+import (
+	"encoding/json"
+	log "github.com/Sirupsen/logrus"
+	"github.com/quintilesims/layer0/api/backend/ecs"
+	"github.com/quintilesims/layer0/common/models"
+	"time"
+)
+
+var DeleteTaskSteps = []Step{
```

Comment:
this was missing previously, when we deleted an `environment` before, `tasks` were not also cleared